Microsoft.HealthVault/Thing/ThingBase.cs

```diff
@@ -835,7 +835,9 @@ public string Serialize()
             XDocument newDocument = SDKHelper.SafeLoadXml(GetItemXml());
 
             string dataName = "thing/data-xml";
-            fetchedDocument.Element(dataName).ReplaceWith(newDocument.Element(dataName));
```

Comment:
Element("path") doesn't support "/" characters, in other words you are selecting an specific Element by name. When this was written, using thing/data-xml was meant to work as a xpath. xpathselectelement is more appropriate for this. 

Could you be more specific on what you mean by: " looks like if there is a problem then updates wont happen .. but that's not the case .. we are able to update .." I am not sure I understood what you mean.  

---
In reply to: [121809660](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/57#discussion_r121809660) [](ancestors = 121809660)